src/components/flow/routers/response/helpers.ts

```diff
@@ -12,20 +13,91 @@ import { getType } from 'config/typeConfigs';
 import { Router, RouterTypes, SwitchRouter, Wait, WaitTypes } from 'flowTypes';
 import { RenderNode } from 'store/flowContext';
 import { NodeEditorSettings, StringEntry } from 'store/nodeEditor';
+import { SelectOption } from 'components/form/select/SelectElement';
+
+export enum AutomatedTestCaseType {
+  AUTO_GENERATED,
+  USER_GENERATED
+}
+
+export interface AutomatedTestCase {
+  type: AutomatedTestCaseType;
+  testText: string;
+  actualCategory: string;
+  confirmedCategory: string;
+  categoriesMatch: boolean;
+  confirmed: boolean;
+}
 
 interface ConfigRouter {
   spell_checker?: boolean;
   spelling_correction_sensitivity?: string;
+  test_cases?: AutomatedTestCase[];
```

Comment:
Just a comment: I'll need to check afterward whether would be needed to change on GoFlow as well to implement this key.